tests/ztest/mock/testcase.yaml

```diff
@@ -1,6 +1,5 @@
 tests:
   testing.ztest.mock:
-    arch_exclude: arm
```

Comment:
Care to tell us why?

Reply:
Yes, sorry, it looks like way back when ztest's mock was written it wasn't compatible with ARM. At the TSC meeting today the one primary concern was that FFF wasn't compatible with ARM. It seems to be building and running on the arm qemu so I wanted to enable the test.

Reply:
Thanks.  Just for the note.  I run a few on my board with Cortex M3 as well.  So it seems working on ARM.